_posts/2022-06-08-packaging-functions-with-helm.md

```diff
@@ -0,0 +1,261 @@
+---
+title: "Packaging OpenFaaS functions with Helm"
+description: "Learn how to package and deploy your OpenFaaS functions with Helm."
+date: 2022-06-08
+image: /images/2022-06-packaging-functions-with-helm/background.png
+categories:
+- helm
+- functions
+- kubernetes
+- enterprise
+author_staff_member: han
+---
+
+Learn how to package and deploy your OpenFaaS functions with Helm.
+
+## Introduction
+We got some questions from our users who asked us how they can provide configuration options when they deploy functions. This can be useful if you want to update the tag of a function or set common configuration values like limits or environment variables across functions without having to maintain multiple versions of your function stack files. This post will give you an overview of the different approaches that are available.
```

Comment:
tag => `image tag for a new version of a function`

Reply:
> This can be useful if you want to update the tag of a function or set common configuration values like limits or environment variables across functions without having to maintain multiple versions of your function stack files.

Split into two sentences.

> This post will give you an overview of the different approaches that are available.

Break onto a new paragraph

In this post we'll explore how to package functions with Helm, and a simpler alternative too.